### PR TITLE
attempt to resolve complaints while loading fixtures

### PR DESCRIPTION
**Pull request by aeischeid**
attempting to deal with some issues encountered using plugin version linked to at https://github.com/gpc/fixtures/pull/8

### CODE REVIEW

**Comment by aeischeid** [comment]
maybe would want to change getDomainClass method in FixtureBeanPostProcessor to use persistantEntity as well ??

``` groovy 
PersistentEntity getDomainClass(clazz) {
	grailsDomainClassMappingContext.getPersistentEntity(clazz.name)
}

File: src/main/groovy/grails/plugin/fixtures/builder/processor/FixtureBeanPostProcessor.groovy [hunk 1]
```diff
@@ -134,17 +135,23 @@ class FixtureBeanPostProcessor implements BeanPostProcessor {
...
+			return isOwningSide(superDomainClass.getPropertyByName(property.name))
 		}
 	}
 
+	PersistentEntity getPersistentEntity(clazz) {
+		grailsDomainClassMappingContext.getPersistentEntity(clazz.name)
+	}
+
 	def getDomainClass(clazz) {
 		grailsApplication.getDomainClass(clazz.name)
```
Comment (aeischeid) [inline src/main/groovy/grails/plugin/fixtures/builder/processor/FixtureBeanPostProcessor.groovy]: if removed `getDomainClass` in favor of `getPersistentEntity` could also do with out wiring in `grailsApplication` in this class at least

**Comment by aeischeid** [comment]
This is in pretty good shape in my tests locally in a multi project build (ie inline plugin) @double16  let us know what you think!

**Comment by double16** [comment]
Any thoughts of fixing the integration tests?

**Comment by aeischeid** [comment]
I can't get them to run properly for me locally. @jpalmer1026 said he got them to go and they passed for him.

**Comment by double16** [comment]
Thanks for the help on this!

**Pull request merged**